packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -37,6 +37,7 @@ export class HandleNotFoundError extends CustomError {
   }
 }
 
+/** New stake keys derived for multi-delegation were not found in the rewardAccounts provider */
```

Comment:
missplaced comment. Should be above OutOfSyncRewardAccounts.
I'll move it after I get more feedback